supabase/functions/brain_v1_1/index.ts

```diff
@@ -0,0 +1,112 @@
+// Follow this setup guide to integrate the Deno language server with your editor:
+// https://deno.land/manual/getting_started/setup_your_environment
+// This enables autocomplete, go to definition, etc.
+
+import { serve } from 'https://deno.land/std@0.168.0/http/server.ts'
+import { supabase } from './_shared/supabaseClient.ts'
+import { corsHeaders } from './_shared/cors.ts'
+
+console.log('Hello from Functions!')
+
+serve(async req => {
+  console.log(req.method)
+
+  if (req.method === 'POST') {
+    const body = await req.json()
+    const yt_url = body.yt_url as string
+    const desired_language = body.desired_language as string
+    const api_key = body.api_key as string
+    const yt_id = yt_url.split('v=')[1]
+
+    // make queue
+    const { data: queue_data, error: queue_error } = await supabase
```

Comment:
Does it check if the video is already in the queue?

Reply:
no. but remember: on the FE if captions exist you return them, otherwise realtime waits until then do exist. in that sense, queue is no longer needed. i could take this out 🤔

Reply:
Queue is no longer needed? What do you mean by that? Shouldn't we add all requested videos in a queue, and then process them asynchronously and notify the extension whenever a new subtitle is ready?

Reply:
correct. your realtime checker is checking if the captions exist yet in our table "captions", it's not checking the queue table. also, since we're executing the transcrlations in a serverless function it will scale and be able to handle multiple requests. we don't need to worry about queueing like before with google app engine.

Reply:
So if i understand correctly, it is the realtime checker which is checking the existence of captions?